### PR TITLE
feat: add maintenance operator helm doc

### PR DESCRIPTION
**Pull request by rollandf**
image

### CODE REVIEW

**Review by e0ne — APPROVED** [review]

File: docs/customizations/helm.rst [hunk 1]
```diff
@@ -289,7 +293,7 @@ SR-IOV Network Operator Helm chart customization options can be found `here <htt
      -
    * - sriov-network-operator.images.operator
      - string
-     - `"nvcr.io/nvidia/mellanox/sriov-network-operator:network-operator-24.10.0"`
+     - `"nvcr.io/nvstaging/mellanox/sriov-network-operator:network-operator-24.10.0-rc.1"`
```
Comment (heyvister1) [inline docs/customizations/helm.rst:296]: Why was this changes?

File: docs/customizations/helm.rst [hunk 2]
```diff
@@ -305,15 +309,15 @@ SR-IOV Network Operator Helm chart customization options can be found `here <htt
      -
    * - sriov-network-operator.images.sriovConfigDaemon
      - string
-     - `"nvcr.io/nvidia/mellanox/sriov-network-operator-config-daemon:network-operator-24.10.0"`
+     - `"nvcr.io/nvstaging/mellanox/sriov-network-operator-config-daemon:network-operator-24.10.0-rc.1"`
```
Comment (heyvister1) [inline docs/customizations/helm.rst:312]: Same as above

File: docs/customizations/helm.rst [hunk 3]
```diff
@@ -305,15 +309,15 @@ SR-IOV Network Operator Helm chart customization options can be found `here <htt
...
      -
    * - sriov-network-operator.images.sriovDevicePlugin
      - string
      - `"ghcr.io/k8snetworkplumbingwg/sriov-network-device-plugin:v3.8.0"`
      -
    * - sriov-network-operator.images.webhook
      - string
-     - `"nvcr.io/nvidia/mellanox/sriov-network-operator-webhook:network-operator-24.10.0"`
+     - `"nvcr.io/nvstaging/mellanox/sriov-network-operator-webhook:network-operator-24.10.0-rc.1"`
```
Comment (heyvister1) [inline docs/customizations/helm.rst:320]: Same as above

File: docs/customizations/helm.rst [hunk 4]
```diff
@@ -123,7 +127,7 @@ General Parameters
      - Init container image version.
    * - operator.repository
      - string
-     - `"nvcr.io/nvidia/cloud-native"`
+     - `"nvcr.io/nvstaging/mellanox"`
```
Comment (heyvister1) [inline docs/customizations/helm.rst:130]: Why was this changed?

File: docs/customizations/helm.rst (hunk 1 above)
Comment (rollandf) [reply docs/customizations/helm.rst:296]: This is what configured in master branch, the generation is done automatically.

**Review by heyvister1 — APPROVED** [review]
